### PR TITLE
Added schemas for useragent parsing

### PR DESCRIPTION
**Pull request by AALEKH**

### CODE REVIEW

File: schemas/com.snowplowanalytics.snowplow/ua_parser/useragent_utils_config/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "http://json-schema.org/schema#",
+	"description": "Schema for useragent utility enrichment",
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/ua_parser/useragent_utils_config/jsonschema/1-0-0]: -> "Schema for configuration of user-agent-utils enrichment"

File: schemas/com.snowplowanalytics.snowplow/ua_parser/useragent_utils_config/jsonschema/1-0-0 [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/ua_parser/useragent_utils_config/jsonschema/1-0-0]: ^^ This path is wrong. Can we `git mv` to:

schemas/com.snowplowanalytics.snowplow/user_agent_utils_config/jsonschema/1-0-0

File: schemas/com.snowplowanalytics.snowplow/ua_parser/ua_parser_config/jsonschema/1-0-0 [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/ua_parser/ua_parser_config/jsonschema/1-0-0]: ^^ This path is wrong. Can we `git mv` to:

    schemas/com.snowplowanalytics.snowplow/ua_parser_config/jsonschema/1-0-0

File: schemas/com.snowplowanalytics.snowplow/ua_parser/useragent_utils_config/jsonschema/1-0-0 (hunk 2 above)
Comment (AALEKH) [inline schemas/com.snowplowanalytics.snowplow/ua_parser/useragent_utils_config/jsonschema/1-0-0]: Sure, will be done

**Comment by AALEKH** [comment]
All Done !!

**Comment by alexanderdean** [comment]
Looks good!

File: schemas/com.snowplowanalytics.snowplow/ua_parser_context/jsonschema/1-0-0 [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+		"osMajor": {
+			"type": "number"
+		},
+		"osMinor": {
+			"type": "number"
+		},
+		"osPatch": {
+			"type": "number"
+		},
+		"ospatchMinor": {
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/ua_parser_context/jsonschema/1-0-0]: Typo @AALEKH - this should be `osPatchMinor`...

File: schemas/com.snowplowanalytics.snowplow/ua_parser_config/jsonschema/1-0-0 [hunk 5]
```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "http://json-schema.org/schema#",
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/ua_parser_config/jsonschema/1-0-0]: Schema should be "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",

**Comment by alexanderdean** [comment]
Thanks @AALEKH - this is merged into release/60 - closing...

**Pull request closed without merging**